helm_deploy/laa-estimate-eligibility/values/ccq-uat.yaml

```diff
@@ -42,6 +42,8 @@ ingress:
   hosts:
     - host: main-check-client-qualifies-legal-aid-uat.cloud-platform.service.justice.gov.uk
       secret: false
+    - host: ccq-dashboard-staging.cloud-platform.service.justice.gov.uk
```

Comment:
As you're now setting up the metabase hosts directly in ingress.yaml, can you delete it from the lists of hosts here and in ccq-staging.yaml?